ms-patches/unknown-unwind-opcodes.yaml

```diff
@@ -0,0 +1,7 @@
+title: Cleanup unknown unwind opcode (0xB) for windows
+- work_item: 2430162
+- author: dhanalla

```

Comment:
Do we need a jbs_bug field or is that a deprecated field?

Reply:
I am not sure if it is deprecated, I just copied this template from other ms-patch branches (https://github.com/microsoft/openjdk-jdk17u/blob/ms-patches/gettemppath2/ms-patches/get-temp-path2.yaml, https://github.com/microsoft/openjdk-jdk17u/blob/ms-patches/large-pages/ms-patches/large-pages.yaml)